### PR TITLE
Fixes PaloAltoNetworks/pandevice#96 - Fixes PaloAltoNetworks/pandevice#84 - Fixes PaloAltoNetworks/ansible-pan#44

### PR DESCRIPTION
**Pull request by shinmog**
Fixes PaloAltoNetworks/pandevice#96 - Fixes PaloAltoNetworks/pandevice#84 - Fixes PaloAltoNetworks/ansible-pan#44

### CODE REVIEW

**Comment by shinmog** [comment]
Live tests against 8.0 pass successfully.

**Comment by shinmog** [comment]
Oops, `panorama.Template.refreshall(pano)` isn't getting the template's children.  Now I remember why I had to use ParentAwareXpath to fix this.  It was for refreshing, not just for correct xml documents for set/edit.

File: pandevice/panorama.py [hunk 1]
```diff
@@ -78,23 +79,109 @@ def _setup(self):
...
+        mode: (6.1 and lower) Can be fips, cc, or normal (default: normal)
+        vpn_disable_mode (bool): (6.1 and lower) VPN disable mode
+
+    """
+    ROOT = Root.DEVICE
+    SUFFIX = ENTRY
+    CHILDTYPES = (
+        "device.Vsys",
+    )
+
```
Comment (btorresgil) [inline pandevice/panorama.py]: What about all the non-vsys objects?  They should probably be allowed children of Template. For example, ManagementProfiles would be a potential child of Template.

**Comment by shinmog** [comment]
The codacy issue is not a real issue (that's not a password).

**Pull request merged**